### DISCUSSION TITLE
G-watch update vibration

### DISCUSSION THREAD

**Discussion by Boneunit** [trupici/G-Watch-Wear#33] [Q&A]
Hi,
Every time there is a blood sugar update, the watch vibrates.
Can we remove this vibration?
Thanks for your help

**Comment by trupici** [comment]
Hi,

in watch face configuration (on the watch) scroll right to Alarms page and
at the very bottom there is the option to enable/disable new data
notifications.
https://github.com/trupici/G-Watch-Wear/wiki/Digital-Watch-Face-Configuration#watch-alarms-configuration

Juraj

Dňa ne 11. 2. 2024, 9:12 Boneunit ***@***.***> napísal(a):

**Comment by Boneunit** [comment]
Hi,
It works great !
Thanks
